src/utils/openai.ts

```diff
@@ -60,8 +60,9 @@ const deduplicateMessages = (array: string[]) => Array.from(new Set(array));
 
 const getPrompt = (locale: string, diff: string) => `Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything, the response must be in the language ${locale}:\n${diff}`;
 
-const model = 'text-davinci-003';
-const encoder = encodingForModel(model);
+const model = 'gpt-3.5-turbo';
+// TODO: update for the new gpt-3.5 model
+const encoder = encodingForModel('text-davinci-003');
```

Comment:
Was waiting for the new one to be released but I guess there's no point in blocking the feature.